### PR TITLE
Update MetaPhlAn2 package for GitHub migration 

### PR DESCRIPTION
**Pull request by fbeghini**

### CODE REVIEW

**Comment by fbeghini** [comment]
@BiocondaBot please add label

File: recipes/metaphlan2/meta.yaml [hunk 1]
```diff
@@ -1,13 +1,13 @@
 {% set name = "metaphlan2" %}
-{% set version = "2.96.1" %}
-{% set sha256 = "4b52aa83716725867b8a5e5f50aa0f73c0bb6ae28e68ca64effe0540b81b32d4" %}
+{% set version = "2.8" %}
```
Comment (bgruening) [inline recipes/metaphlan2/meta.yaml:2]: mh, this is adding an old version? Is that correct?

**Comment by fbeghini** [comment]
Yes, that's right. 2.8 should be the latest working version of MetaPhlAn2

----

Francesco Beghini

PhD Student


Lab. of Computational Metagenomics

Department of Cellular, Computational and Integrative Biology - CIBIO

University of Trento
Via Sommarive 9, 38123 Trento, Italy

Il Lun 27 Apr 2020, 19:49 Björn Grüning <notifications@github.com> ha
scritto:

**Comment by dpryan79** [comment]
Just FYI, 2.96.1 will still be installed instead of this once we merge it. Is there a reason that 2.96 went away after the migration to github?

**Comment by fbeghini** [comment]
We decided to change the name of the software by dropping the trailing "2" in MetaPhlAn2 and use just MetaPhlAn. Version 2.96 has been changed to MetaPhlAn 3.0 alpha

**Comment by dpryan79** [comment]
@bioconda-bot please fetch artifacts

**Comment by dpryan79** [comment]
@bioconda-bot please merge

**Comment by bgruening** [comment]
Mh, but the name is still metaphlan2 -- shouldn't it be metaphlan then?

**Comment by fbeghini** [comment]
Yes, 2.8 is kept under the name metaphlan2 just for reproducibility. The newer versions are under metaphlan

**Pull request merged**